src/DssSpell.sol

```diff
@@ -39,13 +43,63 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    address immutable internal ESM            = DssExecLib.getChangelogAddress("MCD_ESM");
-    address immutable internal RWA_LIQ_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+    uint256 internal constant FOUR_NINE_PCT_RATE = 1000000001516911765932351183;
+
+    uint256 internal constant MILLION = 10**6;
+    uint256 internal constant RAD     = 10**45;
 
     function actions() public override {
 
-        // ----- Additional ESM authorization fix -----
-        DssExecLib.authorize(RWA_LIQ_ORACLE, ESM);
+        uint256 Art;
+        uint256 rate;
+        uint256 line;
+        uint256 maxLineReduction;
+        uint256 totalDebt;
+        VatLike vat = VatLike(DssExecLib.vat());
+
+        // ---------------- RWA008-A Off-boarding Phase 0 ----------------
+        // Forum: https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/51
+
+        // Set RWA008-A Debt Ceiling to 0:
+        (Art, rate, , line, ) = vat.ilks("RWA008-A");
+        maxLineReduction += line;
+        totalDebt += Art * rate;
+        DssExecLib.setIlkDebtCeiling("RWA008-A", 0);
```

Comment:
I explained this in a top-level comment in the PR, but the TL;DR is that we don't want to do that because the current debt for all ilks (except `RWA008-A`) is not zero.